nvidia legacy 390 build failure with Linux 5.10

**Describe the bug**
Build fails with
```
build/kernel/common/inc/nv-linux.h:164:10: fatal error: linux/ioctl32.h: No such file or directory
  164 | #include <linux/ioctl32.h>          /* register_ioctl32_conversion()    */
      |          ^~~~~~~~~~~~~~~~~
```

**To Reproduce**
Steps to reproduce the behavior:
1. build https://gist.github.com/lourkeur/e0a3df6e742d8e105c27ccbc187cf86b

**Expected behavior**
build should succeed.

**Screenshots**

**Additional context**
updating the nvidia package should fix this as [release notes from last week](https://www.nvidia.co.uk/Download/driverResults.aspx/173291/en-uk) state:
>Fixed a driver installation failure on Linux kernel 5.11 release candidates, where the NVIDIA kernel module failed to build with error "fatal error: asm/kmap_types.h: No such file or directory".
PR on the way

**Notify maintainers**
@baracoder 

**Metadata**
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.
 - system: `"x86_64-linux"`
 - host os: `Linux 4.19.187, NixOS, 21.05.20210425.1fe6ed3 (Okapi)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.4pre20210326_dd77f71`
 - channels(root): `""`
 - channels(louis): `""`
 - nixpkgs: `/nix/store/fkjfrpn7yhjwaqrqfp8ifwgfiixc2lxg-source`


Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
# a list of nixos modules affected by the problem
module:
```

--- a/pkgs/os-specific/linux/nvidia-x11/default.nix
+++ b/pkgs/os-specific/linux/nvidia-x11/default.nix
@@ -56,11 +56,11 @@ rec {
 
   # Last one supporting x86
   legacy_390 = generic {
-    version = "390.138";
-    sha256_32bit = "0y3qjygl0kfz9qs0rp9scn1k3l8ym9dib7wpkyh5gs4klcip7xkv";
-    sha256_64bit = "0rnnb5l4i8s76vlg6yvlrxhm2x9wdqw7k5hgf4fyaa3cr3k1kysz";
-    settingsSha256 = "0ad6hwl56nvbdv9g85lw7ywadqvc2gaq9x6d2vjcia9kg4vrmfqx";
-    persistencedSha256 = "15jciyq6i3pz1g67xzqlwmc62v3xswzhjcqmfcdndvlvhcibsimr";
+    version = "390.143";
+    sha256_32bit = "AelrdTTeo/3+ZdXK0iniZDB8gJUkeZQtNoRm25z+bQY=";
+    sha256_64bit = "tyKqcPM71ErK8ZZHLPtxmgrWzv6tfEmxBRveCSwTlO8=";
+    settingsSha256 = "EJPXZbxZS1CMENAYk9dCAIsHsRTXJpj473+JLuhGkWI=";
+    persistencedSha256 = "FtlPF3jCNr18NnImTmr8zJsaK9wbj/aWZ9LwoLr5SeE=";
   };
 
   legacy_340 = generic {

